be/src/exec/vectorized/hdfs_scan_node.cpp

```diff
@@ -243,12 +246,13 @@ Status HdfsScanNode::_create_and_init_scanner(RuntimeState* state, const HdfsFil
     scanner_params.min_max_conjunct_ctxs = _min_max_conjunct_ctxs;
     scanner_params.min_max_tuple_desc = _min_max_tuple_desc;
     scanner_params.hive_column_names = &_hive_column_names;
-    scanner_params.parent = this;
     scanner_params.open_limit = hdfs_file_desc.open_limit;
+    scanner_params.profile = &_profile;
 
     HdfsScanner* scanner = nullptr;
     if (hdfs_file_desc.hdfs_file_format == THdfsFileFormat::PARQUET) {
-        _parquet_profile.init(_runtime_profile.get());
+        _profile.parquet_profile = _pool->add(new HdfsParquetProfile());
```

Comment:
Will this create duplicate `HdfsParquetProfile` while only the last one is in use?

Reply:
RIght. I'll fix that.